helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -104,8 +104,87 @@ receivers:
             service.instance.id:
               enabled: false
           {{- else }}
-          metrics_path: '`"prometheus.io/path" in annotations ? annotations["prometheus.io/path"] : "/metrics"`'
-          endpoint: '`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090`'
+          config:
+            scrape_configs:
+              - job_name: 'istio'
+                metrics_path: '`"prometheus.io/path" in annotations ? annotations["prometheus.io/path"] : "/metrics"`'
+                scrape_interval: 10s
+                static_configs:
+                  - targets: ['`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090`']
```

Comment:
From testing, `9090` as the default port returns more results than what we have in [istio discovery](https://github.com/signalfx/splunk-otel-collector/blob/4bf9ff53d15faef23424ac028e5a0f1c0cf23eb8/cmd/discoverybundler/metadata/receivers/istio.yaml#L15) with `15090`. The large majority of pods have the annotation though, so this usually doesn't make a difference.